### ISSUE TITLE
PORT env support

### ISSUE THREAD

**Issue by matti** [docker-library/ghost#231]
I'm surprised that PORT is still not supported - it's common convention in Heroku or in Google Cloud Run.

https://github.com/docker-library/ghost/issues/177

**Comment by pascalandy** [comment]
Has mentioned https://hub.docker.com/_/ghost, you can do

```
docker run -d --name some-ghost -e url=http://localhost:3001 -p 3001:2368 ghost
```

Is there something else to this??

**Comment by matti** [comment]
Yes, I want to change the :2368 part - heroku expects you to listen in $PORT and so does cloud run

**Comment by pascalandy** [comment]
I don't understand what you are looking for. It looks like a question for your heroku stack. I don't think it's related to this docker image.

**Comment by matti** [comment]
let me try again,

`docker run --port 8080:8080 --env PORT=8080 ghost`

I'm looking for a way to change the internal listening port to something else.

In heroku PORT variable value changes on every boot.

**Comment by tianon** [comment]
Try `server__port=8080` instead (yes, two underscores).

**Comment by matti** [comment]
Thanks! the docs aren't too clear about this.

**Comment by tianon** [comment]
In this case, the relevant section is "Configuration" which links to https://ghost.org/docs/concepts/config/#running-ghost-with-config-env-variables, which is where I gleaned/synthesized `server__port` from (https://ghost.org/docs/concepts/config/#server).
